docker_test.go

```diff
@@ -443,18 +523,117 @@ func TestWebhook(t *testing.T) {
 	require.Equal(t, gjson.GetBytes(body, "anonymousId").Str, "anon-id-new")
 	require.Equal(t, gjson.GetBytes(body, "userId").Str, "identified user id")
 	require.Equal(t, gjson.GetBytes(body, "rudderId").Str, "daf823fb-e8d3-413a-8313-d34cd756f968")
-	require.Equal(t, gjson.GetBytes(body, "type").Str, "track")
-
-
-	// TODO: Verify in Live Evets API
+	require.Equal(t, gjson.GetBytes(body, "type").Str, "identify")
+	
 }
 // Verify Event in POSTGRES
 func TestPostgres(t *testing.T) {
 	var myEvent Event
 	require.Eventually(t, func() bool {
-		eventSql:= "select anonymous_id, user_id from example.tracks limit 1"
+		eventSql:= "select anonymous_id, user_id from example.identifies limit 1"
 		db.QueryRow(eventSql).Scan(&myEvent.anonymous_id, &myEvent.user_id)
 		return myEvent.anonymous_id == "anon-id-new"
 	}, time.Minute, 10*time.Millisecond)
 	require.Equal(t, "identified user id", myEvent.user_id)
+	
+	}
+// Verify Event in Redis	
+func TestRedis(t *testing.T) {
+	fmt.Println(address)
+	conn, err := redigo.Dial("tcp", address)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer conn.Close()
+	require.Eventually(t, func() bool {
+		// Similarly, get the trait1 and convert it to a string.
+		event, _ := redigo.String(conn.Do("HGET", "user:identified user id", "trait1"))
+		return event == "new-val"
+	}, time.Minute, 10*time.Millisecond)
+
+}
+func TestKafka(t *testing.T) {
+	config := sarama.NewConfig()
+	config.ClientID = "go-kafka-consumer"
+	config.Consumer.Return.Errors = true
+
+	brokers := []string{"localhost:9092"}
+
+	// Create new consumer
+	master, err := sarama.NewConsumer(brokers, config)
+	if err != nil {
+		panic(err)
 	}
+	// defer func() {
+	// 	if err := master.Close(); err != nil {
+	// 		panic(err)
+	// 	}
+	// }()
+	topics, _ := master.Topics()
+
+	consumer, errors := consume(topics, master)
+
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, os.Interrupt)
+
+	// Count how many message processed
+	msgCount := 0
+	// Get signnal for finish
+	doneCh := make(chan struct{})
+	
+	go func() {for {
+		select {
+		case msg := <-consumer:
+			msgCount++
+			fmt.Println("Received messages", string(msg.Key), string(msg.Value))
+			require.Equal(t, "identified user id", string(msg.Key))
+			require.Contains(t, string(msg.Value), "new-val")
+			require.Contains(t, string(msg.Value), "identified user id")
+			master.Close()
+		case consumerError := <-errors:
+			msgCount++
+			fmt.Println("Received consumerError ", string(consumerError.Topic), string(consumerError.Partition), consumerError.Err)
+			doneCh <- struct{}{}
+		case <-signals:
+			fmt.Println("Interrupt is detected")
+			doneCh <- struct{}{}
+		}
```

Comment:
I think is better to use a timeout instead of `<-signals`.

```suggestion
		case <-time.After(time.Minute):
             panic("timeout waiting on kafka message")
		}
```